packages/sdk/src/NomadContext.ts

```diff
@@ -173,6 +173,43 @@ export class NomadContext extends MultiProvider<config.Domain> {
     return this.mustGetCore(this.governor.domain);
   }
 
+  async process(txId: string): Promise<ContractTransaction>{
+    const isProduction = this.conf.environment === 'production'
+    const nomadAPI = isProduction
+      ? 'https://bridge-indexer.prod.madlads.tools/tx/'
+      : 'https://bridge-indexer.dev.madlads.tools/tx/'
+    const s3URL = isProduction
```

Comment:
have we discussed S3 proofs as being okay to serve externally?

Reply:
I'll check

Reply:
We already do for self-proving, no?

Reply:
Yep, it's already being served externally from the gui (https://github.com/nomad-xyz/nomad-app/blob/development/src/config/index.ts#L19)